exercises/00-init/src/modules/users/users-saga.js

```diff
@@ -0,0 +1,35 @@
+import { call, fork, put, takeEvery } from 'redux-saga/effects';
+
+import {getUsers as getUsersEffect, addUser as addUserEffect} from 'modules/users/users-effects'
+import UsersActions from 'modules/users/users-actions'
+
+
+function* getUsers() {
+    try {
+        const users = yield call(getUsersEffect);
+        yield put({ type: UsersActions.Types.LOADED, payload: users });
+    }
+    catch (e)
+    {
+        console.log(`getUsers Saga failed:${e}`)

```

Comment:
Prefer `console.error` for logging errors. Not only does it make the message red, but if you pass it an error object, it will print the stack trace as well. So in this case I would probably do 
```
console.error('getUsers saga failed.')
console.error(e)
```